packages/now-next/src/utils.ts

```diff
@@ -300,6 +300,34 @@ export function getDynamicRoutes(
     return [];
   }
 
+  try {
+    // Use routes-manifest if available so that we don't have
```

Comment:
Unnecessary comment, let's remove this and add a comment _below_ for the old implementation.
```
// FALLBACK:
// When `routes-manifest.json` does not exist (old Next.js versions), we'll try to
// require the methods we need from Next.js' internals.
//
// TODO: implement this branch behind a Next.js version check so we don't "fallback"
// to this behavior blindly.
```